python/cuml/tests/test_benchmark.py

```diff
@@ -182,16 +182,30 @@ def predict(self, X):
 # skipping UMAP-Supervised due to issue
 # https://github.com/rapidsai/cuml/issues/4243
 @pytest.mark.parametrize(
-    "algo_name", ["DBSCAN", "LogisticRegression", "ElasticNet", "FIL"]
+    "algo_name",
+    [
+        "DBSCAN",
+        "LogisticRegression",
+        "ElasticNet",
+        "FIL",
+        "xgboost-classification",
+        "xgboost-regression",
+    ],
 )
 def test_real_algos_runner(algo_name):
     pair = algorithms.algorithm_by_name(algo_name)
 
-    if algo_name == "FIL":
+    if algo_name in ["FIL", "xgboost-classification", "xgboost-regression"]:
         pytest.importorskip("xgboost")
```

Comment:
This means we skip FIL tests when xgboost is not available?

Reply:
yup, that was already true before this PR, to benchmark fil we build and train an xgboost model that then we infer with FIL (example [here](https://github.com/rapidsai/cuml/blob/8882188d0ca6ac06eb5ef357477d083f63b48d44/python/cuml/cuml/benchmark/bench_helper_funcs.py#L124))